test/unit/jobs/cleanup-test.js

```diff
@@ -103,6 +110,39 @@ describe("Delete unused files", () => {
     }, 200);
   });
 
+  it("should delete financial data that has not been accessed within the last 24 hours", (done) => {
+    // Wait for file to be deleted before running test.
+    setTimeout(function() {
+      cleanupJob.financial.findByKey("risefinancial_realtime_HXXHCTR5AQFQ_-Kn5MaUE2cpYN-FCQSOU_1M_", (err, foundFinancial) => {
+        expect(Object.keys(foundFinancial.data).length).to.equal(0);
+
+        done();
+      })
+    }, 200);
+  });
+
+  it("should delete spreadsheet data that has not been accessed within the last 24 hours", (done) => {
+    // Wait for file to be deleted before running test.
+    setTimeout(function() {
```

Comment:
It'll still be async with a promise or a callback.  If you don't care about the return value it can be ignored.  In general, having to resort to timeouts in a unit test is an antipattern, imo.